e2e/tests/init_test.go

```diff
@@ -854,62 +864,119 @@ func (s *suite) getApplicationInfo(appName string) *apptypes.Application {
+func (s *suite) getApplicationTransferEndHeight(accName string) int64 {
+	application := s.getApplicationInfo(accName)
+	require.NotNil(s, application.GetPendingTransfer())
+
+	sharedParams := s.getSharedParams()
+
+	return apptypes.GetApplicationTransferHeight(&sharedParams, application)
+}
+
+// getServiceComputeUnitsPerRelay returns the compute units per relay for a given service ID
+func (s *suite) getServiceComputeUnitsPerRelay(serviceId string) uint64 {
 	args := []string{
 		"query",
-		"shared",
-		"params",
+		"service",
+		"show-service",
+		serviceId,
 		"--output=json",
 	}
 
 	res, err := s.pocketd.RunCommandOnHostWithRetry("", numQueryRetries, args...)
 	require.NoError(s, err, "error getting shared module params")
 
-	var resp sharedtypes.QueryParamsResponse
+	var resp servicetypes.QueryGetServiceResponse
 	responseBz := []byte(strings.TrimSpace(res.Stdout))
 	s.cdc.MustUnmarshalJSON(responseBz, &resp)
-	unbondingHeight := apptypes.GetApplicationUnbondingHeight(&resp.Params, application)
-	return unbondingHeight
+	return resp.Service.ComputeUnitsPerRelay
 }
 
-// getApplicationTransferEndHeight returns the height at which the application will be transferred to the destination.
-func (s *suite) getApplicationTransferEndHeight(accName string) int64 {
-	application := s.getApplicationInfo(accName)
-	require.NotNil(s, application.GetPendingTransfer())
-
+// getSharedParams returns the shared module parameters
+func (s *suite) getSharedParams() sharedtypes.Params {
 	args := []string{
 		"query",
 		"shared",
 		"params",
 		"--output=json",
 	}
+	res, err := s.pocketd.RunCommandOnHost("", args...)
+	require.NoError(s, err, "error querying shared params")
 
-	res, err := s.pocketd.RunCommandOnHostWithRetry("", numQueryRetries, args...)
-	require.NoError(s, err, "error getting shared module params")
+	var sharedParamsRes sharedtypes.QueryParamsResponse
 
-	var resp sharedtypes.QueryParamsResponse
-	responseBz := []byte(strings.TrimSpace(res.Stdout))
-	s.cdc.MustUnmarshalJSON(responseBz, &resp)
+	s.cdc.MustUnmarshalJSON([]byte(res.Stdout), &sharedParamsRes)
+	require.NoError(s, err)
 
-	return apptypes.GetApplicationTransferHeight(&resp.Params, application)
+	return sharedParamsRes.Params
 }
 
-// getServiceComputeUnitsPerRelay returns the compute units per relay for a given service ID
-func (s *suite) getServiceComputeUnitsPerRelay(serviceId string) uint64 {
+// getCurrentBlockHeight returns the current block height
+func (s *suite) getCurrentBlockHeight() int64 {
 	args := []string{
 		"query",
-		"service",
-		"show-service",
-		serviceId,
+		"block",
 		"--output=json",
 	}
+	res, err := s.pocketd.RunCommandOnHost("", args...)
+	require.NoError(s, err, "error querying for the latest block")
+
+	// Remove the first line of the response to avoid unmarshalling non JSON data
+	// since the query block command returns "Falling back to latest block height:"
+	// as the first line when no height is provided.
```

Comment:
```suggestion
	// Remove the first line of the response to avoid unmarshalling non JSON data.
	// This is needed because, when no height is provided, the query block command returns:
	// 	"Falling back to latest block height:"
```